docs/Getting-Started-with-Balance-Ball.md

```diff
@@ -83,27 +88,28 @@ environment around the Agents.
 
 ### Brain
 
-The Ball3DBrain GameObject in the scene, which contains a Brain component, is a
-child of the Academy object. (All Brain objects in a scene must be children of
-the Academy.) All the Agents in the 3D Balance Ball environment use the same
-Brain instance. A Brain doesn't store any information about an Agent, it just
+Brains are assets that exist in your project folder. The Ball3DAgents are connected
+to a brain, for example : the **Ball3DBrain**.
+A Brain doesn't store any information about an Agent, it just
 routes the Agent's collected observations to the decision making process and
 returns the chosen action to the Agent. Thus, all Agents can share the same
 Brain, but act independently. The Brain settings tell you quite a bit about how
 an Agent works.
 
-The **Brain Type** determines how an Agent makes its decisions. The **External**
-and **Internal** types work together — use **External** when training your
-Agents; use **Internal** when using the trained model. The **Heuristic** Brain
-allows you to hand-code the Agent's logic by extending the Decision class.
+There are 3 types of brains, you can create new ones by selecting `Assets -> 
+Create -> ML-Agents -> Brain`.
+The **Learning Brain** is a brain that uses a Neural Network to take decisions.
+When the Brain is checked as `Train` in the Academy **Training Hub**, the 
+Neural Network will be trained by the external process. You can also use the
```

Comment:
This is somewhat misleading, since there isn't a NN in Unity being trained, but the NN lives outside Unity.